### PR TITLE
support ClientSession context manager

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

**Comment by mosquito** [comment]
Please add some tests for new API.

File: aiohttp_xmlrpc/client.py [hunk 1]
```diff
@@ -110,5 +110,11 @@ def method(*args, **kwargs):
 
         return method
 
+    def __aenter__(self):
+        return type(self.client).__aenter__(self.client)
```
Comment (mosquito) [inline aiohttp_xmlrpc/client.py]: I feel it's might be shorter and simpler. Like this:
```python
return self.client.__aenter__()
```

Comment (graingert) [reply aiohttp_xmlrpc/client.py]: But that's not how these methods are called by `async with`

Comment (mosquito) [reply aiohttp_xmlrpc/client.py]: But my method generates less bytecode and provides similar result.

**Pull request merged**